### PR TITLE
8-1-Incognito

### PR DESCRIPTION
**Pull request by shubhamkumar-revefi**
HTTP Server programmed in c++

### CODE REVIEW

File: task-2/advance/submissions/Incognito/Server.cpp [hunk 1]
```diff
@@ -0,0 +1,181 @@
+#include <bits/stdc++.h>
+#include <string.h>
+#include <sys/types.h>
+#include <winsock2.h>
+#include <unistd.h>
+#include <ws2tcpip.h>
+#include <boost/lexical_cast.hpp>
```
Comment (abhinavkgrd) [inline task-2/advance/submissions/Incognito/Server.cpp]: @scheleon  We were asked to used only build-in C libararies.

**Comment by shubhamkumar-revefi** [comment]
Boost:: lexical_cast was just used to add the strings up in order to store the data in a single string. Well I can change it, sprintf will work fine if you insist. And bits/stdc++ was used just to save the time so that I won't have  to include each header file at once.

**Comment by abhinavkgrd** [comment]
> 
> 
> Boost:: lexical_cast was just used to add the strings up in order to store the data in a single string. Well I can change it, sprintf will work fine if you insist.

Oo Soory, I didn't knew that.

**Comment by abhinavkgrd** [comment]
@scheleon I am not a member of ACM. I was just curious about your implementation so, i looked into the code.
You can contact Ayush Sinha Bhaiya to check if its ok to use Boost libraries.

**Comment by sarthakgupta072** [comment]
> @scheleon I am not a member of ACM. I was just curious about your implementation so, i looked into the code.
> You can contact Ayush Sinha Bhaiya to check if its ok to use Boost libraries.

Nice going @Abhinav-grd ! Actually, this is how open source works. People learn from each other.

**Pull request merged**